website/docs/guides/core-to-cloud-2.md

```diff
@@ -0,0 +1,196 @@
+
+</expandable>
+
+<expandable alt_header="Analyze current workflows, review processes, and team structures">
+
+Discover how dbt Cloud can help simplify development, orchestration, and testing.
+- **Development**: Develop dbt models, allowing you to build, test, run, and version control your dbt projects using the dbt Cloud CLI (command line interface or code editor) or dbt Cloud IDE (browser-based).
+- **Orchestration**: Create custom schedules to run your production jobs. Schedule jobs by day of the week, time of day, or a recurring interval.
+  - Set up CI to ensure developer effectiveness, and CD jobs to deploy changes as soon as they’re merged. 
+  - Link deploy jobs together by [triggering a job](/docs/deploy/deploy-jobs#trigger-on-job-completion) when another one is completed. 
+  - For the most flexibility, you can use the [dbt Cloud API](https://docs.getdbt.com/dbt-cloud/api-v2#/) to trigger jobs. This makes sense when you want to integrate dbt execution with other data workflows.
+- **Continuous integration (CI)**: Run your dbt projects in a temporary schema when new commits are pushed to open pull requests. This build-on-PR functionality is a great way to catch bugs before deploying to production.
+  - For many teams, dbt Cloud CI represents a major improvement compared to their previous development workflows.
+- **How are you defining tests today?**: While testing production data is important, it’s not the most efficient way to catch logical errors introduced by developers You can use [unit testing](/docs/build/unit-tests) to allow you to validate your SQL modeling logic on a small set of static inputs *before* you materialize your full model in production.
+
+</expandable>
+
+<expandable alt_header="Understand access control">
+
+Transition to dbt Cloud's [access control](/docs/cloud/manage-access/about-user-access) mechanisms to ensure security and proper access management. dbt Cloud administrators can use dbt Cloud's permission model to control user-level access in a dbt Cloud account:
+- **License-based access controls:** Users are configured with account-wide license types. These licenses control the things a user can do within the application: view project metadata, develop changes within those projects, or administer access to those projects.
+- **Role-based Access Control (RBAC):** Users are assigned to *groups* that have specific permissions on specific projects or all projects in the account. A user may be a member of multiple groups, and those groups may have permissions on multiple projects.
+
+</expandable>
+
+<expandable alt_header="Manage environments"> 
+
+If you require isolation between production and non-production data environments due to sensitive data, dbt Cloud can support Development, Staging (_soon_), and Production data environments.
+
+This provides developers with the benefits of an enhanced workflow while ensuring isolation between Staging and Production data, and locking down permissions on Prod.
```

Comment:
I don't think we really treat production and staging as proper nouns elsewhere in the docs. Is there a particular reason for doing so here?

Reply:
i _think_ this is related to what's coming soon in terms of the diff environments. going to loop @jtcohen6 as i thought it was related which is why i kept it capitalized.

Reply:
@matthewshaver This is related to the new "Staging" type of deployment environment that we're rolling out alongside "Production! I think you're driving that one ([internal notion](https://www.notion.so/dbtlabs/dbt-Cloud-staging-environment-684abeb5baf24b6fbd55bd5c9f56b606?pvs=31)). Agree that proper nouns make more sense here.